### DISCUSSION TITLE
Actively changing target latency

### DISCUSSION THREAD

**Discussion by Aroeloe** [f1multiviewer/issue-tracker#406] [Feature requests]
During this weekend the sync was a bit. I understand this was because something was changed on the F1TV side.
During the race and quali i tried to sync everything, by syncing the live timing to my main screen. And syncing my onboards to the live timing by pausing the onboard on the apex off a recognisable corner in such as 1, 6, 10 and 18 and unpausing when the blips on track map when through the corner. But this only changed the actual latency and not the target latency so the player undid my adjustment everytime. 
So my feature request is that the pause button during live watching affects both of the target latency and the actual latency.

**Comment by jeppeolesen** [comment]
Hey there @Aroeloe 

Some updates have been made to the sync API since this ticket was filed, and should work much better than it has in the past.

I'll close this request due to it's age. If you still experience issues please file a bug report here: https://github.com/f1multiviewer/issue-tracker/issues

You can also read more about how the sync works in the newer versions of multiviewer here: https://multiviewer.dev/docs/usage/syncing-streams
